### ISSUE TITLE
Adding global function is not reflected to Google Spread Sheet by clasp push to Apps Script bounded with Google Spread Sheet

### ISSUE THREAD

**Issue by yumetodo** [google/clasp#108]
## Abstruct

Adding global function is not reflected to Google Spread Sheet by clasp push to Apps Script bounded with Google Spread Sheet.

## Expected Behavior

![img](http://www.plantuml.com/plantuml/svg/TP1DQWCn38NtSmfT81UOHKdfeWUO4nYF9ZQr9QCVsjo-nKbnMfoDmUZf-z0xgmKnhsMv27k3M88sQAvf0O3Rz_Jopdmf-DgQRb5oitKa_ez-mriJ3FkM4EqjQC8LY2qVLpYSecKce51nGThlPfCh64CCfG3XvyWTmlMkWHiLEdRe7sNF_J5X_01BED3ubTLq3kfRfmbk_Sius6cV8sR5zViU0VCMfrv1So4GL2_Mgpl_F5xBDm00)

## Actual Behavior

![img](http://www.plantuml.com/plantuml/svg/nP91RiGW34NtSuf_WRb05jLKNVG0EG459w0XDWAZQMvV2Qf98z5TfIxznN_VDhuLDLdh5YzhbffWeoa9gHP_0T3Bwykxo1hfDQKosnoIJlq5iVksQTwX0cjY1DC3IsMhGHXBbWtTZZbbCWwp9z96xtgNczgIpjARAPucM4_s3lLqeEap52rZK7DT1xW9B8exouDHglLFt8zXN2qASa4X3E4Oc5ef-GWxbl5hqoqgBFlHCPkDyB7_GS2JRRtXbVvjcsV68fNT636ueNOj9y1m-d5k9gsPaQdKgEtlRiIkRl4B)

(note: this image will not be the actual routine.)

## Steps to Reproduce the Problem

  1. Write new global function in local
  2. `clasp push` to Apps Script bounded with Google Spread Sheets
  3. Open the Google Spread Sheets
  4. call the new function
  5. Not known such function
  6. Open Online Script Editor of the Google Spread Sheets
  7. make some dust edit and **save**.
  8. call the new function
  9. You can call the function.

## Specifications

  - Node version (`node -v`): `v9.11.1`
  - Version (`npm list -g | grep clasp`): `+-- @google/clasp@1.1.5`
  - OS (Mac/Linux/Windows): Windows

**Comment by grant** [comment]
How are you (4) calling  the new function?

Did you say `@customfunction` in your method comment?
Can you give a specific example?

**Comment by yumetodo** [comment]
>How are you (4) calling the new function?
>
>Did you say @customfunction in your method comment?

Yes. I'm trying to create customfunction via clasp.

**Comment by yumetodo** [comment]
https://github.com/miraiken/event-201803-analyse

This is final production code.

To satisfy Google Apps Script's requirement, I use [gas-webpack-plugin](https://github.com/fossamagna/gas-webpack-plugin).

**Comment by grant** [comment]
I don't think this is `clasp`'s fault.
Please link the exact function you can't call.

**Comment by yumetodo** [comment]
>I don't think this is clasp's fault.

Who has the responsibility of this issue, you think?

>Please link the exact function you can't call.

By Step (7), now, I can call correctly.

![image](https://user-images.githubusercontent.com/10869046/38695981-d4ba1088-3ec8-11e8-891f-7b81b0ccc5b7.png)

**Comment by grant** [comment]
I'm saying, I think if you create the same script without `clasp`, you'll have the same problem.
Can you try that and report here if not using `clasp` creates the same effect?

**Comment by grant** [comment]
Were you able to try to reproduce the error without clasp?

**Comment by yumetodo** [comment]
>I'm saying, I think if you create the same script without clasp, you'll have the same problem.

If that is true, how you explain step 7 effect?

**Comment by grant** [comment]
Step 7 states: `make some dust edit and save.`
I don't understand the question. There are many errors with the English sentences that make it hard to figure out what error you're facing.

I just added a customfunction to a bounded Sheets script without opening the script.google.com editor and successfully called a custom function.

1. i.e. create a local file like this:

```js
/**
 * @customfunction
 */
function myFunction() {
  return 4;
}
```

2. Then `clasp push`.
3. Then call `myFunction()` in Google Sheets. You'll see the value 4.

The expected behavior in the issue here is not valid or is not reproducible. Create a video if you are still experiencing this behavior without any solution. Provide code samples. Closing as not reproducible.
